tests/unit/plugins/modules/test_kubevirt_vm.py

```diff
@@ -141,6 +199,24 @@ def k8s_module_params_create(module_params_create, vm_definition_create):
     }
 
 
+@pytest.fixture(scope="module")
+def k8s_module_params_running(module_params_running, vm_definition_running):
+    return module_params_running | {
+        "generate_name": None,
```

Comment:
Extract to `module_params_default`?

Reply:
Not possible, this would fail validation of params when instatiating the k8s module where we set `name` and `generate_name` to be mutually exclusive.

Reply:
Got it. Thanks for the clarification.